jdaviz/configs/cubeviz/plugins/viewers.py

```diff
@@ -36,6 +36,9 @@ def __init__(self, *args, **kwargs):
         self._subscribe_to_layers_update()
         self.state.add_callback('reference_data', self._initial_x_axis)
 
+        # Hide axes by default
+        self.state.show_axes = False
```

Comment:
I wonder if this part at least deserves a change log entry?

Reply:
Change log added.

Reply:
Just in case anyone wants it the old way, could we add a note in the Cubeviz docs in case people want to see it the old way (with axes)? The user could call:
```python
for viewer_name in ['flux-viewer', 'uncert-viewer']:
    cubeviz.app.get_viewer(viewer_name).state.show_axes = True
```

Reply:
Should we also update the Cubeviz docs screenshots and animations?

Reply:
(and also a note that it can be toggled via the UI in plot options)

Reply:
> Should we also update the Cubeviz docs screenshots and animations?

That should be its own ticket. I think way more than that is outdated by now. 😬 

> Just in case anyone wants it the old way

I think we agreed that `.app` isn't public API, so I would rather not advertise that. But I can mention the Plot Options.

Reply:
Added a commit to update Plot Option docs. I think only Cubeviz is affected.